proto/ibc/core/client/v1/query.proto

```diff
@@ -195,6 +200,20 @@ message QueryClientParamsResponse {
   Params params = 1;
 }
 
+// QueryClientCreatorRequest is the request type for the Query/ClientCreator RPC
+// method.
+message QueryClientCreatorRequest {
```

Comment:
Should this be in v2?

Reply:
The keys set/get/delete are in v1 keeper which is why i left the query here as well